CMakeLists.txt

```
@@ -8,6 +8,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 set (sources
 	main.cpp
+	oc.h
```

Headers are included, implementation is compiled.
```suggestion
	oc.cpp
```